### ISSUE TITLE
Could not detect the SDK! 

### ISSUE THREAD

**Issue by Ares-03** [axonasif/gearlock#41]
Describe the bug:- 
He @axonasif , @electrikjesus , After installing Gearlock I am facing this problem and I tried to find some solution on the internet, but I did not find any official solution to this problem on the internet.  So can you tell why this is happening or can you suggest a solution to this problem.  I am facing this problem mostly with all Android OS.  And this old pc of mine is running windows 10 64 bit smoothly.

**Desktop (please complete the following information):**
 - OS: Android-x86 A8.1, A9, A7... 
 - CPU: Pentium Dual core E5500 2ghz
 - BIOS: Legacy( Mbr)

![PXL_20231228_021150211.jpg](https://github.com/axonasif/gearlock/assets/106290878/7702b77e-34eb-4b38-a24a-fa7ba62a9777)

Please help....

**Comment by Ares-03** [comment]
> Describe the bug:- 
> He @axonasif , @electrikjesus , After installing Gearlock I am facing this problem and I tried to find some solution on the internet, but I did not find any official solution to this problem on the internet.  So can you tell why this is happening or can you suggest a solution to this problem.  I am facing this problem mostly with all Android OS.  And this old pc of mine is running windows 10 64 bit smoothly.
> 
> **Desktop (please complete the following information):**
>  - OS: Android-x86 A8.1, A9, A7... 
>  - CPU: Pentium Dual core E5500 2ghz
>  - BIOS: Legacy( Mbr)

And yeah i want to say this Android x86 is running in 32gb (3.0)  pendrive 
> 
> ![PXL_20231228_021150211.jpg](https://github.com/axonasif/gearlock/assets/106290878/7702b77e-34eb-4b38-a24a-fa7ba62a9777)
> 
> Please help.... 
> 
>

**Comment by axonasif** [comment]
Inside a pendrive?

And by Android-x86 do you mean https://www.android-x86.org/ ?

**Comment by Ares-03** [comment]
> Inside a pendrive?
> 
> And by Android-x86 do you mean https://www.android-x86.org/ ?

Yeah this one and this one https://archive.org/details/androidtv-x86

Do you have any solution or explanation why this is happening?
